src/client/components/ProfileCareerResult/ProfileCareerResult.styles.css

```diff
@@ -0,0 +1,51 @@
+.robo_text {
```

Comment:
As Niels mentioned in class, the names should be kebab case => robot-text.
and the same goes for all